### ISSUE TITLE
Missing Module "PowerPass.Common.ps1" from Release v1.6.1

### ISSUE THREAD

**Issue by zedtran** [chopinrlz/powerpass#4]
Module [PowerPass.Common.ps1](https://github.com/chopinrlz/powerpass/blob/main/module/PowerPass.Common.ps1) is missing from [Zip archive](https://github.com/chopinrlz/powerpass/releases/download/v1.6.1/PowerPass-1.6.1.zip) of [Release v1.6.1](https://github.com/chopinrlz/powerpass/releases/tag/v1.6.1). 

Script `Deploy-PowerPass.ps1` execution results in the following output:
```
PS  ."...\PowerPass\PowerPass-1.6.1\Deploy-PowerPass.ps1"
Please indicate the edition of PowerPass you want to deploy: (1) AES or (2) DP API with KeePass support? [1/2]: 1
Install path: C:\Users\...\Documents\WindowsPowerShell\Modules\PowerPass
Copy-Item : Cannot find path '...\PowerPass-1.6.1\module\PowerPass.Common.ps1' because it does not exist.
At ...\PowerPass-1.6.1\Deploy-PowerPass.ps1:184 char:26
+ ...        $itemsToDeploy | Copy-Item -Destination $targetLocation -Force
+                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (...\Po...Pass.Common.ps1:String) [Copy-Item], ItemNotFoundException
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.CopyItemCommand

WARNING: Missing file: C:\Users\...\Documents\WindowsPowerShell\Modules\PowerPass\PowerPass.Common.ps1
PowerPass deployed with warnings, please review messages above
```

Following steps in the [deployment verification guidance](https://chopinrlz.github.io/powerpass/deployment#step-3-verify-the-deployment), the following module import error occurred:

```
PS ...\PowerPass> Import-Module PowerPass
. : The term 'C:\Users\...\Documents\WindowsPowerShell\Modules\PowerPass\PowerPass.Common.ps1' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the
name, or if a path was included, verify that the path is correct and try again.
At C:\Users\...\Documents\WindowsPowerShell\Modules\PowerPass\PowerPass.psm1:46 char:3
+ . ($PowerPass.CommonSourcePath)
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\...Pass.Common.ps1:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

Manual copy of module `PowerPass.Common.ps1` into the unpacked `module` download folder and re-running `Deploy-PowerPass.ps1` appeared successful.

**Comment by chopinrlz** [comment]
Thanks for catching this zedtran, I've resolved this in the latest release 1.6.2

**Comment by zedtran** [comment]
No problem! Happy to be of help!
